websites/D/Desu-Online/metadata.json

```
@@ -20,5 +20,6 @@
 		"anime"
 	],
 	"iframe": true,
-	"iFrameRegExp": ".*"
+	"iFrameMatches": ["*://*.cda.pl/*", "*://*.google.com/*", "*://*.mega.nz/*", "*://*.dailymotion.com/*", "*://*.sibnet.ru/*", "*://*.fileditchstuff.me/*", "*://*.yourupload.com/*", "*://*.vk.com/*", "*://*.wolfstream.tv/*", "*://*.ok.ru/*"],
```

if you're not sure if only those domains above work change it to this
```suggestion
	"iFrameRegExp": ".*",
	"iFrameMatches": ["*://*/*"],
```

otherwise if you are sure change it to this:
```suggestion
	"iFrameRegExp": ".*",
	"iFrameMatches": ["*://*.cda.pl/*", "*://*.google.com/*", "*://*.mega.nz/*", "*://*.dailymotion.com/*", "*://*.sibnet.ru/*", "*://*.fileditchstuff.me/*", "*://*.yourupload.com/*", "*://*.vk.com/*", "*://*.wolfstream.tv/*", "*://*.ok.ru/*"],
```